InWorldz/InWorldz.VivoxVoice/VivoxVoiceModule.cs

```diff
@@ -744,7 +744,7 @@ private string RegionGetOrCreateChannel(Scene scene, LandData land)
             string landName;
             string parentId;
 
-            lock (m_parents) parentId = m_forcedChannelName == String.Empty ? m_parents[scene.RegionInfo.RegionID.ToString()] : m_parents[m_forcedChannelName];
+            lock (m_parents) parentId = String.IsNullOrEmpty(m_forcedChannelName) ? m_parents[scene.RegionInfo.RegionID.ToString()] : m_parents[m_forcedChannelName];
```

Comment:
No semantic change: The only assignment to `m_forcedChannelName` is guaranteed to not be null: line 147 `m_forcedChannelName = m_config.GetString("forced_channel_name", String.Empty).ToLower();`